### PR TITLE
Makes ffmpeg and fullscreen configurable.

### PR DESCRIPTION
**Pull request by abaire**

### CODE REVIEW

File: xemutest/test_base.py [hunk 1]
```diff
@@ -87,6 +107,8 @@ def prepare_config(self):
 			f.write(config)
 
 	def launch_video_capture(self):
+		if not self.test_env.video_capture_enabled:
+			return
```
Comment (mborgerson) [inline xemutest/test_base.py:111]: ffmpeg path is not used to launch ffmpeg here

Comment (abaire) [reply xemutest/test_base.py:111]: Sorry, had this commit split off of my larger pgraph tests PR and forgot to merge down later fixes.

File: xemutest/test_base.py [hunk 2]
```diff
@@ -20,6 +20,33 @@
...
+		cur_dir = os.getcwd()
+		if not xemu_path:
+			if platform.system() == 'Windows':
+				self.xemu_path = os.path.join(cur_dir, 'xemu.exe')
+			else:
+				self.xemu_path = 'xemu'
+		else:
+			self.xemu_path = xemu_path
+
+		self.ffmpeg_path = ffmpeg_path
```
Comment (mborgerson) [inline xemutest/test_base.py:42]: If `ffmpeg_path` isn't provided, let's use `ffmpeg[.exe]` (resolved by environment PATH)

Comment (mborgerson) [reply xemutest/test_base.py:42]: nvm, I see you changed it below

**Pull request merged**